Client/src/components/common/date-picker/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+function Datepicker() {
+  return (
```

Comment:
you need a new line after the import statement line 2 
change the function name to  DatePicker

Reply:
and remember to change the function name in the export

Reply:
i fixed it :+1: